deploy/images/db/download.sql

```diff
@@ -57,106 +39,84 @@ DECLARE
      rid  INTEGER;
 BEGIN
 
+     -- Find the file
      SELECT * INTO archive_rec FROM local_ega.archive_files WHERE stable_id = sid LIMIT 1;
 
      IF archive_rec IS NULL THEN
-     	RAISE EXCEPTION 'Archive file not found for stable_id: % ', sid;
+     	RAISE EXCEPTION 'archive file not found for stable_id: % ', sid;
```

Comment:
maybe this?
```suggestion
     	RAISE EXCEPTION 'archived file not found for stable_id: % ', sid;
```

Reply:
Sure, will update the code and push it today